tuned/profiles/functions/function_intel_recommended_pstate.py

```diff
@@ -0,0 +1,44 @@
+import os
+import tuned.logs
+from . import base
+from tuned.profiles.exceptions import InvalidProfileException
+
+log = tuned.logs.get()
+processor_name = ("sandybridge",
+        "ivybridge",
+        "haswell",
+        "broadwell",
+        "skylake")
+
+pmu_path = "/sys/devices/cpu/caps/pmu_name"
+active = "active"
+disable = "disable"
+
+class intel_recommended_pstate(base.Function):
+    """
+    Checks the processor code name and return the pstate
```

Comment:
I would copy some part of the PR description to the file docstring too. Especially the section explaining the latency reasons for pre-IceLake vs. post-IceLake.

Reply:
Done